controlplane/kubeadm/internal/controllers/helpers_test.go

```diff
@@ -338,7 +338,7 @@ func TestCloneConfigsAndGenerateMachine(t *testing.T) {
 			},
 		},
 	}
-	g.Expect(env.Create(ctx, genericInfrastructureMachineTemplate)).To(Succeed())
+	g.Expect(env.CreateAndWait(ctx, genericInfrastructureMachineTemplate)).To(Succeed())
```

Comment:
I think I had to change this because now the tests are also using cached unstructured get/list calls (production code in KCP always did)